### ISSUE TITLE
simplify module

### ISSUE THREAD

**Issue by derhuerst** [hemanth/life-expectancy#3]
Thanks for this module! It is part of a greater effort to make interesting data truly machine-readable & versioned.

This request is rather general, trying to reduce the module to the bare minimum of size (including dependencies) & complexity.

As this module is only supposed to contain the life expectancy data for all countries, it is not its job to parse human country names like "India". Instead, I'd prefer it to use [ISO 3166-1-alpha-2 codes](https://github.com/fannarsh/country-list). Users can parse "India" by themselves.

I'd therefore prefer to set the `main` field to `life.json`. `life.json` could be generated from the raw data in a `npm run build` step, an object with the ISO codes as keys.

**Comment by hemanth** [comment]
Internally I am using the `country-list` module. 

`main` field to `life.json` was my first thought too.

**Comment by derhuerst** [comment]
Exactly. My point is to remove it from the dependencies and introduce a build step that make `life.json` contain ISO contry codes.

**Comment by hemanth** [comment]
So we would need both `code` and `name` in the new `life.json`

**Comment by derhuerst** [comment]
Putting in `name` is a design choice. I can easily get the name from the ISO code using other modules,  it's up to you on how much additional work consumers will need to do. It's a compromise between comfort and module size imo.

I propose to organise `life.json` as an object, with each key being an ISO contry code, and each value being an object containing life expectancy etc.

**Comment by hemanth** [comment]
> I propose to organise life.json as an object, with each key being an ISO contry code.

SGTM

**Comment by derhuerst** [comment]
I'd make a PR. For the build script, I depend on #4.
